### PR TITLE
Always handle events from watchers

### PR DESCRIPTION
**Pull request by dmage**
workqueue.AddRateLimited may start to ignore events when a burst occurs. And we don't want to delay handling of CR changes.

### CODE REVIEW

File: pkg/operator/controller.go [hunk 1]
```diff
@@ -202,42 +202,36 @@ func (c *Controller) sync() error {
 func (c *Controller) eventProcessor() {
 	for {
 		obj, shutdown := c.workqueue.Get()
-
 		if shutdown {
 			return
 		}
 
-		err := func(obj interface{}) error {
+		glog.V(1).Infof("get event from workqueue")
```
[the remark this answers is not in the document]
Comment (dmage) [reply pkg/operator/controller.go]: No, the message "add event to workqueue" has the same log level.

**Comment by legionus** [comment]
```
level=error msg="\t* aws_instance.master.0: Error launching source instance: timeout while waiting for state to become 'success' (timeout: 30s)"
```
/retest

**Comment by legionus** [comment]
Good job, CI !!!
```
level=info msg="Waiting 30m0s for the Kubernetes API..."
level=debug msg="Still waiting for the Kubernetes API: Get https://ci-op-yqnrtzx8-37ae6-api.origin-ci-int-aws.dev.rhcloud.com:6443/version?timeout=32s: dial tcp 23.20.153.105:6443: i/o timeout"
```
/retest

**Pull request merged**